lib/rucio/core/rse_selector.py

```diff
@@ -13,9 +13,10 @@
 # limitations under the License.
 
 from random import shuffle, uniform
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Iterable, Optional
```

Comment:
`Iterable` should be imported from `collections.abc` (see: https://docs.python.org/3/library/typing.html#typing.Iterable)

Reply:
Importing `Iterable` from `collections.abc` now